CRM/Financial/BAO/Order.php

```diff
@@ -179,7 +201,7 @@ public function _unserialize(array $data): void {
   public function getOverrideTotalAmount() {
     // The override amount is only valid for quick config price sets where more
     // than one field has not been selected.
-    if (!$this->overrideTotalAmount || !$this->supportsOverrideAmount() || count($this->getPriceOptions()) > 1) {
```

Comment:
ditching the supportsOverrideAmount because
1) it creates order issues - the prices set needs to be determined before it can be accessed and 
2) it just came out of logic moved from a form - but really I don't think there is any real reason at a BAO level why only quickconfig can support amount overrides. Quick config was always envisaged as a form presentation issue but the implementation got greedy. I think it's a form layer issue